pkg/api/handlers/generic/containers_stats.go

```diff
@@ -151,6 +151,7 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
 						UsageInUsermode:   cgroupStat.CPU.Usage.Total - cgroupStat.CPU.Usage.Kernel,
 					},
 					SystemUsage: 0,
+					CPU: stats.CPU,
```

Comment:
do we need something similar at line 75?

Reply:
I don't think so. Those are previous stats, and I cannot think of usecase where it would make sense have it.